src/app/components/PortraitVideoModal/index.tsx

```diff
@@ -127,10 +127,10 @@ const PortraitVideoModal = ({
 }: PortraitVideoModalProps) => {
   const {
     translations: {
-      media: { closeVideo = 'Close' },
+      media: { closeVideo = 'Close', modalLabel = 'Media modal' },
```

Comment:
Do users know what a modal is? I wonder if there's a better translation e.g. Media Player / Media Player Popup?

Reply:
Going to name it media player for now as its better than Media Modal as a description.